### PR TITLE
Update views.py

### PR DESCRIPTION
**Pull request by vesteinn**
Mimetype changed to content_type in django 1.7.

### CODE REVIEW

File: rosetta/views.py [hunk 1]
```diff
@@ -429,4 +429,4 @@ def translate_text(request):
         except TranslateApiException as e:
             data = {'success': False, 'error': "Translation API Exception: {0}".format(e.message)}
 
-    return HttpResponse(json.dumps(data), mimetype='application/json')
+    return HttpResponse(json.dumps(data), content_type='application/json')
```
Comment (mbi) [inline rosetta/views.py:432]: but, but, … that'll break on Django < 1.7, won't it?

Comment (vesteinn) [inline rosetta/views.py:432]: Nah, this goes far back, see https://github.com/django/django/commit/0afbca9c60

Comment (mbi) [inline rosetta/views.py:432]: You are very right, I stand corrected.

**Comment by mbi** [comment]
Thank you!

**Pull request merged**